tests/services/detectionlists/test_high_risk_employee.py

```diff
@@ -29,40 +27,43 @@ def test_choices_are_correct(self):
 
 class TestHighRiskEmployeeClient(object):
```

Comment:
Check for Client is the wrong spots in these tests... This one should be `TestHighRiskEmployeeService`

Reply:
thanks